### ISSUE TITLE
TfLiteCoreMlDelegateOptions documentation is not up-to-date

### ISSUE THREAD

**Issue by hguihot** [tensorflow/tensorflow#57565]
https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/delegates/coreml

It currently shows the following structure, which is not up to date:
```
typedef struct {
 // We have dummy for now as we can't have empty struct in C.
 char dummy;
} TfLiteCoreMlDelegateOptions;
```

See current definition in https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/delegates/coreml/coreml_delegate.h.

**Comment by mohantym** [comment]
Hi @hguihot !
Thanks for reporting this bug. Made a [PR ](https://github.com/tensorflow/tensorflow/pull/57578)to fix this issue.
Thank you!

**Comment by mohantym** [comment]
@hguihot @Vaani-pathariya!
Marking this issue as resolved as concerned PR #57578 has been merged now.
Thank you!

**Comment by mohantym** [comment]
Are you satisfied with the resolution of your issue?
[Yes](https://goo.gl/forms/Oe0tEvODFRoI2gJF3)
[No](https://goo.gl/forms/fUjzOfrtkFbrOT8d2)
